src/Features/Core/Portable/CodeRefactorings/SyncNamespace/AbstractChangeNamespaceService.cs

```diff
@@ -589,11 +588,11 @@ private static async Task<ImmutableArray<ReferencedSymbol>> FindReferencesAsync(
             string newNamespace,
             CancellationToken cancellationToken)
         {
-            // Can't apply change to certain document, simply return unchanged.
+            // Can't apply change to certain document, returns null to indicate this.
```

Comment:
How much work would it be to null annotate this file so we can have this gotcha documented in the signature itself?

Reply:
I will look into this

Reply:
I have enabled nullable for this method only